src/Cake.Core.Tests/Fixtures/ScriptHostFixture.cs

```diff
@@ -18,9 +19,9 @@ public TestingScriptHost(ICakeEngine engine, ICakeContext context)
             {
             }
 
-            public override CakeReport RunTarget(string target)
+            public override Task<CakeReport> RunTarget(string target)
             {
-                return new CakeReport();
+                return System.Threading.Tasks.Task.FromResult(new CakeReport());
```

Comment:
This is actually necessary because there's a build failure if you don't fully qualify.  `ScriptHost.Task` conflicts.